kinto/config/kinto.tpl

```diff
@@ -11,49 +11,131 @@ port = %(http_port)s
+# kinto.statsd_prefix = kinto-prod
+
+# kinto.http_scheme = https
+# kinto.http_host = kinto.services.mozilla.com
+
+# kinto.backoff = 10
+# kinto.retry_after_seconds = 30
+# kinto.eos =
+
+# Full options list for .ini file
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html
+
+# Feature settings
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#feature-settings
 #
-# Backends.
+# kinto.readonly = False
+# kinto.batch_max_requests = 25
+# kinto.paginate_by=None
+# kinto.<object-type>_id_generator=UUID4 <bucket, collection, group, record>
+# Experimental JSON-schema on collection
+# kinto.experimental_collection_schema_validation = False
 #
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#activating-the-permissions-endpoint
+# kinto.experimental_permissions_endpoint = False
+#
+# kinto.trailing_slash_redirect_enabled = True
+# kinto.heartbeat_timeout_seconds = 10
+
+# Backends
 # https://kinto.readthedocs.io/en/latest/configuration/settings.html#storage
 #
 kinto.storage_backend = {storage_backend}
 kinto.storage_url = {storage_url}
+# kinto.storage_max_fetch_size = 10000
+# kinto.storage_pool_size = 25
+# kinto.storage_max_overflow = 5
+# kinto.storage_pool_recycle = -1
+# kinto.storage_pool_timeout = 30
+# kinto.storage_max_backlog = -1
+
+# Cache
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#cache
+#
 kinto.cache_backend = {cache_backend}
 kinto.cache_url = {cache_url}
+# kinto.cache_prefix = ''
 # kinto.cache_max_size_bytes = 524288
+# kinto.cache_pool_size = 25
+# kinto.cache_max_overflow = 5
+# kinto.cache_pool_recycle = -1
+# kinto.cache_pool_timeout = 30
+# kinto.cache_max_backlog = -1
+
+# Permissions.
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#permissions
+#
 kinto.permission_backend = {permission_backend}
 kinto.permission_url = {permission_url}
+# kinto.permission_pool_size = 25
+# kinto.permission_max_overflow = 5
+# kinto.permission_pool_recycle = 1
+# kinto.permission_pool_timeout = 30
+# kinto.permission_max_backlog - 1
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#bypass-permissions-with-configuration
+# kinto.bucket_create_principals = system.Authenticated
 
+# Scheme, host, and port
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#scheme-host-and-port
 #
-# Features.
+# kinto.http_host = None
+# kinto.http_scheme = None
+
+# Logging and Monitoring
 #
-# kinto.readonly = false
-# kinto.bucket_create_principals = system.Authenticated
-# kinto.batch_max_requests = 25
+# https://kinto.readthedocs.io/en/latest/configuration/settings.html#logging-and-monitoring
+# kinto.statsd_backend = kinto.core.statsd
+# kinto.statsd_prefix = kinto
+# kinto.statsd_url = None
+# kinto.newrelic_config = None
+# kinto.newrelic_env = dev
 
-# Experimental JSON-schema on collection
-# kinto.experimental_collection_schema_validation = true
+# Logging configuration
```

Comment:
If possible I would keep the Logging configuration at the end of the file because they change the INI sections while all into settings should be in the same `[app:main]` one.